app/controllers/api/v1/merchants/revenue_controller.rb

```diff
@@ -1,26 +1,23 @@
 class Api::V1::Merchants::RevenueController < ApplicationController
-  include ActionView::Helpers::NumberHelper  
-  respond_to :json
-  
+
   def index
-    @merchants = Merchant.order_by_revenue(params[:quantity].to_i)
+    @merchants = Merchant.order_by_revenue(params[:quantity])
```

Comment:
**I found out `.to_i` here isn't necessary. Apparently ActiveRecord handles this automatically.**